server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2229,11 +2234,11 @@ public void testSnapshotRelocatingPrimary() throws Exception {
         assertThat(client.prepareSearch("test-idx").setSize(0).get().getHits().getTotalHits().value, equalTo(100L));
 
         logger.info("--> start relocations");
-        allowNodes("test-idx", internalCluster().numDataNodes());
+        allowNodes("test-idx", 1);
 
         logger.info("--> wait for relocations to start");
 
-        waitForRelocationsToStart("test-idx", TimeValue.timeValueMillis(300));
+        assertTrue(waitForRelocationsToStart("test-idx", TimeValue.timeValueMinutes(1)));
```

Comment:
I think you could profitably inline `waitForRelocationsToStart` (it's only used here) and convert it to an `assertBusy`.

Reply:
Yea, inlined it :)